hrms/payroll/doctype/salary_structure/salary_structure.json

```diff
@@ -48,7 +48,6 @@
    "search_index": 1
   },
   {
-   "fetch_from": "company.default_letter_head",
```

Comment:
Instead, you can let this be and just enable "fetch if empty". That will let you overwrite the letter head

Also let's enable "Allow on Submit" for this field so that people don't have to cancel the entire structure to edit this.

Reply:
Smart, shoulda thought of that, done